app/helpers/application_helper.rb

```diff
@@ -183,4 +183,11 @@ def get_facility_locations_by_visn(visn)
     end
     location_list
   end
+
+  def ga_private?
```

Comment:
Nit: I think we can rename this to be a bit more generic, in case we need to use it for something else in the future. Maybe `is_path_private?` or something.

Reply:
I would also consider writing a small test for this method 😄

Reply:
👍 I ended up writing a feature test for this instead of unit tests. 

🤔 I hear what you're saying about potentially reusing this method down the line. I'd be nervous about someone accidentally reusing this in a view where authentication should be done in the controller instead. I added a little comment to the method that links to the DAP implementation page.